sale_order_product_recommendation/tests/test_recommendation.py

```diff
@@ -127,6 +127,21 @@ def test_recommendations_ordered_by_code(self):
             ],
         )
 
+    def test_recommendations_with_note(self):
```

Comment:
What do you do in this test?

Reply:
Generate the recomendations with the wizard in a sale order with notes. It is what gives an error without the fix